rubyUtils.gradle

```diff
@@ -158,10 +158,16 @@ void rake(File projectDir, File buildDir, String task) {
         jruby.currentDirectory = projectDir
         jruby.runScriptlet("require 'rake'; require 'time'")
         jruby.runScriptlet("""
-                rake = Rake.application
-                rake.init
-                rake.load_rakefile
-                rake['${task}'].invoke
+                begin
+                    rake = Rake.application
+                    rake.init
+                    rake.load_rakefile
+                    rake['${task}'].invoke
+                rescue => e
+                    puts "Rake task error: #{e.class}: #{e.message}"
+                    puts "Backtrace: #{e.backtrace.join("\\n")}"
```

Comment:
This isnt really helping either as rake itself is shelling out to make 😅 .

Reply:
I could take or leave this change. I dont think its hurting anything, but it may not actually be that useful in practice.